src/jdk.jdeps/share/classes/com/sun/tools/javap/TypeAnnotationWriter.java

```diff
@@ -108,22 +114,22 @@ private void addNote(int pc, Note note) {
     }
 
     @Override
-    void writeDetails(Instruction instr) {
+    void writeDetails(int pc, Instruction instr) {
         String indent = space(2); // get from Options?
-        int pc = instr.getPC();
         List<Note> notes = pcMap.get(pc);
         if (notes != null) {
             for (Note n: notes) {
                 print(indent);
                 print("@");
-                annotationWriter.write(n.anno, false, true);
+                annotationWriter.write(n.anno, false, true, lr);
                 print(", ");
-                println(StringUtils.toLowerCase(n.kind.toString()));
+                println(n.kind.toString().toLowerCase(Locale.US));
             }
         }
     }
 
     private AnnotationWriter annotationWriter;
     private ClassWriter classWriter;
     private Map<Integer, List<Note>> pcMap;
+    private CodeAttribute lr;
```

Comment:
nit: name `lr` doesn't relate much to the type of the field

Reply:
Right, `lr` historically came from label resolver, I'll fix it.